lewis/core/statemachine.py

```diff
@@ -294,6 +295,8 @@ def bind_handlers_by_name(self, instance, override=False, prefix=None):
         """
         if prefix is None:
             prefix = {}
+        if not isinstance(prefix, dict) and hasattr(prefix, '__iter__'):
+            prefix = dict(zip(['on_entry', 'in_state', 'on_exit'], prefix))
 
         # Merge prefix defaults with any provided prefixes
         prefix = dict(list(self._prefix.items()) + list(prefix.items()))
```

Comment:
Actually, it looks like `self._prefix` is only every used in this function... was this always the case? I could have sworn it had more uses.